python/ray/ray_constants.py

```diff
@@ -256,3 +256,7 @@ def to_memory_units(memory_bytes, round_up):
 # Should be kept in sync with kSetupWorkerFilename in
 # src/ray/common/constants.h.
 SETUP_WORKER_FILENAME = "setup_worker.py"
+
+# The default value of the placement group. -1 is used so that None
+# argument can be used for a different purpose.
+PLACEMENT_GROUP_DEFAULT_VAL = -1
```

Comment:
Hmm, why not just use `actor.options(placement_group=None)` as the default arg (which would default to inheritance)? and then use `placement_group=False` to override?

Reply:
That works too. I don't have strong preference in how API looks like (I think we don't need to worry about backward compatbility now). Thoughts @ericl?

Reply:
Please remove this and just use "default" as the string value.